overlord/devicestate/firstboot_test.go

```diff
@@ -77,6 +78,11 @@ type firstBootBaseTest struct {
 func (t *firstBootBaseTest) setupBaseTest(c *C, s *seedtest.SeedSnaps) {
 	t.BaseTest.SetUpTest(c)
 
+	// TODO: temporary: skip due to timeouts on riscv64
+	if runtime.GOARCH == "riscv64" || os.Getenv("SNAPD_SKIP_SLOW_TESTS") != "" {
```

Comment:
could we set SNAPD_SKIP_SLOW_TESTS in debian/rules instead?

Reply:
I think we can, it will be a bit more involved so I'm wondering if it's okay to do it as a followup? 

I was also doing `go test -v ./...` and sorting the times:
```
$ go test -count=1 ./... | awk '/^ok/{print $3" "$2}'  |sort -h |tail -n10
10.594s github.com/snapcore/snapd/asserts
11.184s github.com/snapcore/snapd/cmd/snap-bootstrap
11.841s github.com/snapcore/snapd/image
13.433s github.com/snapcore/snapd/boot
13.774s github.com/snapcore/snapd/overlord/servicestate
13.936s github.com/snapcore/snapd/daemon
15.570s github.com/snapcore/snapd/cmd/snap
18.839s github.com/snapcore/snapd/overlord/devicestate
26.018s github.com/snapcore/snapd/overlord/snapstate
34.173s github.com/snapcore/snapd/overlord
```
so it seems like it would be good to drill this down a bit more, maybe some more need to be marked as "slow" tests. We also can maybe use testflinger to get some more detailed stats on the riscv64.